tests/wasmer-web/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
```

Comment:
This is debatable, but I'd put this under `wasmer-web/` instead of in the integration tests dir.

Reply:
Funnily enough, that's where I was originally putting them. 

A problem you can run into is that we're adding a bunch of extra dependencies just for communicating with a browser (i.e. `fantoccini` which pulls in `hyper`) and those dependencies won't be able to compile to `wasm32-unknown-unknown` with the rest of the crate. That means doing `wasm-pack test` in the `lib/wasi-web` directory won't work without a bunch of `#[cfg]`'s and fiddling around in `Cargo.toml`. 

It felt like putting them next to our other integration tests was the cleanest option.